deployments/engine/docker-compose/1m1e.yaml

```diff
@@ -9,6 +9,7 @@ services:
     command:
       - "/tiflow"
       - "master"
+      - "--name=server-master"
```

Comment:
Why remove the configuration here?

Reply:
"remove" -> "add" ?

The default name is `"server-master-" + $advertise_addr` which is too long for the name.